### ISSUE TITLE
"Rerun simulation" button

### ISSUE THREAD

**Issue by kenthklui** [Twintop/beotorch-issues#4]
Add a button in "Your simulations" that reruns that specific sim with the same settings.

**Comment by Twintop** [comment]
Necroing this bug, looking for some opinions:

Should this button rerun the same sim exactly verbatim as it was run the first time or something slightly different? The uses for slightly different I can think of might include:
- Grab latest armory data, everything else the same.
- Rerun again to get stat weights if it wasn't selected initially.
- Rerun everything but at a different number of iterations.
- Rerun everything but with a different encounter type.
- Rerun everything but with a different encounter length.
- Rerun everything but with a different set of talents.
- Etc....

It might be possible to create a menu where you can choose something like "Rerun with..." and choose some parameter that you want to change.

Thoughts?

**Comment by kenthklui** [comment]
1. Original intent was that using latest armory data was the point, but hey, you could probably make it a checkbox if you wanted: "Use latest armory profile". That said, I have trouble imagining why someone would want to run the exact same sim twice.
2. Probably don't need to change stat weight settings. At that point it would be more like a "Load saved simulation" feature.
3. Don't need a different number of iterations. Same as 2.
4. Not needed. Same as 2.
5. Etc.
   ...

The point of a one-click rerun button would be so that you can immediately compare a new setup with a previous one. Everything else seems more like loading a saved simulation, which would be feature creep.

**Comment by Twintop** [comment]
Makes sense to me. I think the only thing I'll need to add is a way to name the new simulation. Thanks! :+1: 

EDIT: Note to self: need to handle custom talentsets. If a single talentset is selected, allow the user to choose whether to keep that talentset or use whatever is from the armory. If multiple talentsets, use those again.
